script/Ciel/Day 8/08_01_CIEL08_1C.txt

```diff
@@ -327,72 +353,79 @@ I understand how she feels.
 [sha:d0a5528802dddce2ba69f579521ee4f65134298f]{
 -- Page 94, Offset 18486.
 -- しかし、人体というものは時としてよく分からない仕組みで動くものなのです。
-However, humans can be swayed in mysterious ways.
+But, the human body just behaves in unpredictable ways sometimes.
```

Comment:
'mysterious ways' or 'strange ways' feels like the correct collocation

Reply:
mysterious ways